tests/components/uptimerobot/test_binary_sensor.py

```diff
@@ -0,0 +1,82 @@
+"""Test Uptime Robot binary_sensor."""
+
+from unittest.mock import patch
+
+from pyuptimerobot import UptimeRobotAuthenticationException
+
+from homeassistant.components.binary_sensor import DEVICE_CLASS_CONNECTIVITY
+from homeassistant.components.uptimerobot.const import (
+    ATTRIBUTION,
+    COORDINATOR_UPDATE_INTERVAL,
+    DOMAIN,
+)
+from homeassistant.const import STATE_ON, STATE_UNAVAILABLE
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+from homeassistant.util import dt
+
+from .common import (
+    MOCK_UPTIMEROBOT_API_KEY,
+    MOCK_UPTIMEROBOT_MONITOR,
+    UPTIMEROBOT_TEST_ENTITY,
+    MockApiResponseKey,
+    mock_uptimerobot_api_response,
+    setup_uptimerobot_integration,
+)
+
+from tests.common import async_fire_time_changed
+
+
+async def test_config_import(hass: HomeAssistant):
```

Comment:
For completeness.
```suggestion
async def test_config_import(hass: HomeAssistant) -> None:
```

Reply:
There were a few more places, addressed in <[`a03f4b4` ](https://github.com/home-assistant/core/pull/54314/commits/a03f4b4eba54c4afe0943d67b9f334ce4a39120a)>